### PR TITLE
Suggested edits

### PR DESCRIPTION
**Pull request by jayantk**
I'm out of time right now but here's where I got so far. Main suggestion is to move the explanatory content to the "How Express Relay" works header section and then put in some big CTA buttons on the index page that get people to the integration content. I don't actually know how to put in a button but will figure it out later.

Wanted to send this as a PR though to see if you agree with this organization.

Please merge this if you're happy with it

### CODE REVIEW

File: pages/express-relay/how-express-relay-works.mdx [hunk 1]
```diff
@@ -1 +1,44 @@
...
+![](images/express_relay/express_relay_schematic.svg)
+
+The diagram above shows how Express Relay changes the MEV landscape for a liquidation.
+In the status quo (left), Searchers tip miners in order to guarantee that their liquidation transaction lands on-chain.
+Their transaction directly interacts with the protocol exposing the liquidation opportunity, and the liquidation bonus flows back to the Searcher.
+With Express Relay (right), Searchers submit bids for their transaction to the Express Relay auction.
+The auction submits the winning bids to the blockchain, where the transactions are processed by the Express Relay Entrypoint before being forwarded on to the integrated protocol.
+The Express Relay Entrypoint collects payment from the Searchers and forwards a share of the revenue back to the integrated protocol.
+
+FIXME: I think the diagram is wrong (specifically the liquidation bonus going back to express relay)
```
Comment (anihamde) [inline pages/express-relay/how-express-relay-works.mdx:25]: the diagram is wrong, good catch. we should correct this in the final version

File: pages/express-relay/how-express-relay-works.mdx [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+## Which protocols can use Express Relay?
+
+Any protocol with permissionless and valuable operations can use Express Relay.
```
Comment (anihamde) [inline pages/express-relay/how-express-relay-works.mdx:29]: same point about "permissionless"

File: pages/express-relay/how-express-relay-works.mdx [hunk 3]
```diff
@@ -1 +1,44 @@
...
+These operations generate MEV, as the validators control which searchers can access them.
```
Comment (aditya520) [inline pages/express-relay/how-express-relay-works.mdx:30]: These two sentences can be rewritten as "Any protocol exposing valuable operations that generate MEV "opportunities" can use Express Relay

File: pages/express-relay/index.mdx [hunk 4]
```diff
@@ -1,32 +1,16 @@
 # Introduction
 
-Many protocols generate MEV on a regular basis. Borrow-lending protocols for example expose lucrative liquidation bonuses that searchers compete for on tips made to the chain's miners or validators. Most of the value in the liquidation opportunity is captured by the validators, so the liquidation bonus is in essence a transfer of wealth from the protocol to the validators.
+Express Relay is a priority auction that enables protocols to eliminate miner-extractable value (MEV).
 
-Express Relay enables protocols to recapture the MEV they currently leak to validators. By integrating with Express Relay, a protocol internalizes the auction for priority on its lucrative transactions (e.g. liquidations). It also taps into an existing set of searchers already integrated into and actively performing liquidations through Express Relay, reducing the need for the protocol to boostrap and maintain or incentivize its own set of liquidators.
+**For Protocol Developers:** Express Relay’s auction primitive allows your protocol to prioritize access to permissionless operations, eliminating the extractive role of miners in ordering transactions.
```
[the remark this answers is not in the document]
Comment (aditya520) [reply pages/express-relay/index.mdx:5]: IMO, this statement bit overkill for intro page.  @anihamde

File: pages/express-relay/index.mdx [hunk 5]
```diff
@@ -1,32 +1,16 @@
...
+A network of established searchers compete in the auctions, allowing you to avoid spending time and energy bootstrapping your own protocol-specific searcher network.
+Learn how to [Integrate with Express Relay as a protocol](/express-relay/integrate-as-protocol).
 
-![](images/express_relay/express_relay_schematic.svg)
+FIXME: This link above and the ones below should be big buttons
 
-## What is Express Relay?
+**For Searchers:** Express Relay aggregates liquidation and other MEV opportunities across integrated DeFi protocols, providing easy and unified access.
```
[the remark this answers is not in the document]
Comment (aditya520) [reply pages/express-relay/index.mdx:11]: Or we can scrap the world "liquidation" here and write,

Express Relay aggregate _competing_ MEV opportunities across integrated Defi protocols......

**Pull request merged**